--- drupal7/modules/user/user.module
+++ yad7/modules/user/user.module
@@ -1931,13 +1931,14 @@ function user_menu_link_alter(&$link) {
 
 /**
  * Implements hook_translated_menu_link_alter().
- */
+
 function user_translated_menu_link_alter(&$link) {
   // Hide the "User account" link for anonymous users.
   if ($link['link_path'] == 'user' && $link['module'] == 'system' && !$GLOBALS['user']->uid) {
     $link['hidden'] = 1;
   }
 }
+*/
 
 /**
  * Implements hook_admin_paths().
